contrib/kafka-spout/README.md

```diff
@@ -0,0 +1,375 @@
+Storm Kafka
+====================
+
+Provides core Storm and Trident spout implementations for consuming data from Apache Kafka 0.8.x.
```

Comment:
i was wondering do we need to modify README.md to mention users that current release only support `core api`,`Trident api` is not included.